testing/openapi_validation/validators/enum.go

```diff
@@ -102,6 +102,16 @@ var (
 			// GeoIP base constants: GeoIP2-ISP, GeoIP2-ASN, ...
 			"spec.versions[*].schema.openAPIV3Schema.properties.spec.properties.geoIP2.properties.maxmindEditionIDs.items",
 		},
+		"modules/099-ceph-csi/crds/cephcsi.yaml": {
+			// ignore
```

Comment:
Could you please elaborate more on why we need to ignore these errors?

Reply:
Done.